### PR TITLE
Removed unwanted/extra semicolon in script syntax of cfabort

### PR DESCRIPTION
**Pull request by saravanakumar404**
**Removed unwanted/extra semicolon in script syntax of cfabort**

### CODE REVIEW

File: data/en/cfabort.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
 	"name":"cfabort",
 	"type":"tag",
 	"syntax":"<cfabort>",
+	"script":"abort;",
```
Comment (shaedrich) [inline data/en/cfabort.json]: As far as I can see, your changes do not align with your pr's title: There's an additional semicolon,

Comment (saravanakumar404) [reply data/en/cfabort.json]: In cfdocs, cfabort tag script syntax displaying as `abort;;`(having double semicolons) in UI. by using this line I've avoided/ removed extra semicolon. please see the screenshot.
double_semicons

**Comment by KamasamaK** [comment]
This issue causing the extra semicolon is at https://github.com/foundeo/cfdocs/blob/117185026e02672af92fabc725083889eb8ac674/views/doc.cfm#L37-L39

It's appending another `;` even though the `replaceScript` function is already appending a `;`.

**Comment by saravanakumar404** [comment]
> The issue causing the extra semicolon is at
> 
> https://github.com/foundeo/cfdocs/blob/117185026e02672af92fabc725083889eb8ac674/views/doc.cfm#L37-L39
> 
> It's appending another `;` even though the `replaceScript` function is already appending a `;`.

@KamasamaK I can able to remove second semicolon using this line "script":"cfscript;"
please see my screenshot.
image

this is my local result.

**Comment by pfreitag** [comment]
Ok, my thought on this is let's just rely on the `"script"` key and we can remove that bit of logic in the view. So as such I've merged this one.

**Pull request merged**